src/sentry/workflow_engine/models/workflow.py

```diff
@@ -47,3 +49,14 @@ class Meta:
                 fields=["name", "organization"], name="unique_workflow_name_per_org"
             )
         ]
+
+    def evaluate_trigger_conditions(self, evt: GroupEvent) -> bool:
+        """
+        Evaluate the conditions for the workflow trigger and return the results.
+        If there isn't a when_condition_group, the workflow should always trigger.
```

Comment:
it should only trigger if the IF conditions pass too, right? will they be evaluated in this function?

Reply:
yep! i'll update the comment to make it clearer.